src/pricing/calculations/roof.js

```diff
@@ -1,26 +1,31 @@
-import { tiles } from '../content/roof/tiles'
-import { slope } from '../content/roof/slope'
-import { overHang } from '../content/roof/overHang'
+import { tiles } from '../content/roof/categories'
+import { slope } from '../content/roof/categories'
+import { overHang } from '../content/roof/categories'
```

Comment:
You can just do:
`import {tiles, slope, overHang} from '../content/roof/categories'`